spec.md

```diff
@@ -600,6 +612,147 @@ message Volume {
   // the same attributes. This field is OPTIONAL and when present MUST
   // be passed to volume validation and publishing calls.
   map<string,string> attributes = 3;
+
+  // The topological segment(s) the provisioned volume is accessible
+  // from. A topological segment is a sub-division of a cluster, like
+  // region, zone, rack, etc.
+  // Each key must consist of alphanumeric characters, '-', '_' or '.'.
+  // Each value MUST contain 1 or more valid strings.
```

Comment:
What constitutes a "valid" string for a value?

Reply:
This is a good question. I will clarify.